src/storage/sequelize.ts

```diff
@@ -143,14 +143,18 @@ export class SequelizeStorage implements UmzugStorage {
 	async executed(): Promise<string[]> {
 		await this.model.sync();
 		const migrations: any[] = await this.model.findAll({ order: [[this.columnName, 'ASC']] });
-		return migrations.map(migration => {
+		return migrations.reduce((migrations, migration) => {
```

Comment:
Let's not shadow the `migrations` variable. In fact, the linter should protect us from this. Will check why it's not soon.